themes/prism-material-dark.css

```diff
@@ -0,0 +1,220 @@
+code[class*="language-"],
+pre[class*="language-"] {
+  text-align: left;
+  white-space: pre;
+  word-spacing: normal;
+  word-break: normal;
+  word-wrap: normal;
+  hyphens: none;
+  color: inherit;
+  text-shadow: none;
+  font-family: Roboto Mono, monospace;
+  font-size: 1em;
+  line-height: 1.5em;
+  tab-size: 4;
+}
+
+code[class*="language-"]::-moz-selection,
+pre[class*="language-"]::-moz-selection,
+code[class*="language-"] ::-moz-selection,
+pre[class*="language-"] ::-moz-selection {
+  text-shadow: none;
+  opacity: 1;
+  color: #eee;
+  background: #363636;
+}
+
+code[class*="language-"]::selection,
+pre[class*="language-"]::selection,
+code[class*="language-"] ::selection,
+pre[class*="language-"] ::selection {
+  text-shadow: none;
+  opacity: 1;
+  color: #eee;
+  background: #363636;
+}
+
+@media print {
+  code[class*="language-"],
+  pre[class*="language-"] {
+    text-shadow: none !important;
+  }
+}
+
+:not(pre) > code[class*="language-"] {
+  white-space: normal;
+  color: #eee;
+  background: #2f2f2f;
+  border-radius: 0.2em;
+  padding: 0.1em;
+}
+
+pre > code[class*="language-"] {
+  background: none;
+}
+
+pre[class*="language-"] {
+  overflow: auto;
+  position: relative;
+  color: #eee;
+  background: #2f2f2f;
+  border-radius: 0;
+  margin: 0.5em 0;
+  padding: 1.25em 1em;
+}
+
+pre[class*="language-"].language-css > code,
+pre[class*="language-"].language-sass > code,
+pre[class*="language-"].language-scss > code {
```

Comment:
You don't need the `[class*="language-"]` if you're specifying `.language-css` etc. right after

Reply:
Same issue on all themes.

Reply:
You're right, I fixed this.